tests/fixtures/extension-harness.ts

```
@@ -0,0 +1,455 @@
+  people?: unknown[];
+  rosterEntries?: unknown[];
+  lastUpdated?: string | null;
+  currentUserUid?: string | null;
+}
+
+const repoRoot = process.cwd();
+const distDir = path.resolve(repoRoot, 'dist');
+const sampleDataPath = path.resolve(repoRoot, 'src', 'data', 'sample-activities.json');
+
+export const test = base.extend<ExtensionHarness>({
+  context: async ({}, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)